docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/where-syntax.md

```diff
@@ -6,19 +6,20 @@ Currently, the `WHERE` statement only applies to the `GO` statement.
 WHERE (expression [ AND | OR expression ...])  
 ```
 
-Usually, `WHERE` is a set of logical combination that filter vertex or edge properties.
+Usually, `WHERE` is a set of logical combination that filters vertex or edge properties.
 
-> As syntactic sugar, you can freely choose to use both `AND` and `&&`. They are both boolean logical and. So do `OR` and `||`.
+> As syntactic sugar, you can freely choose to use both `AND` and  `&&` as boolean logical and, `OR` and `||` as boolean logical or.
 
 ## Examples
 
 ```ngql
-/* GO FROM 201 OVER like */  -- Apply in a GO statement
-WHERE e1.prop1 >= 17     -- the edge e1's property prop1 is larger than 17
-
-WHERE $^.v1.prop1 == $$.v2.prop2  -- the source vertex v1's property prop1 is equivalent with destination vertex v2's property prop2
-
-WHERE ((e3.prop3 < 0.5) OR ($^.v4.prop4 != "hello")) AND $$.v5.prop5 == "world"   -- logical combination is allowed
-
-WHERE 1 == 1 OR TRUE    --always TRUE
+-- the edge e1's property prop1 is larger than 17
+nebula> GO FROM 201 OVER e1 WHERE e1.prop1 >= 17
+-- the source vertex v1's property prop1 is equivalent with dest vertex v2's property prop2
+nebula> GO FROM 201 OVER e1 WHERE $^.v1.prop1 == $$.v2.prop2
```

Comment:
For a query statement, do you need to add a semicolon? If it is, I think you should put a semicolon for each query statement in this guide.

Reply:
If there are multiple queries, a semicolon must be used, if not, both are ok